aubo_i5:
  #home configuration, the robot end effector pose when all joints are zeros
  #general format [x,y,z,x,y,z,wã€‘
  #the first three numbers are position, the last four numbers are unit quaternion
  home_configuration: [-0.0405, -0.36325, 1.0085,0.500, -0.500, 0.500, 0.500]

  #is the screw axes presented in body frame
  isInBodyFrame: true

  #screw axes
  #general format S1 = [W;V] col vector
  screw_axes:
    S1: [1,0,0,0,-0.36325,0.0405]
    S2: [0,0,1,-0.0405,0.8865,0]
    S3: [0,0,1,-0.0405,0.4085,0]
    S4: [0,0,1,-0.0405,0.1025,0]
    S5: [1,0,0,0,-0.24175,0.0405]
    S6: [0,0,1,-0.0405,0,0]
